### PR TITLE
Adjust encryption key rotation

### PR DESCRIPTION
**Pull request by graetzer**

### CODE REVIEW

File: 3.7/security-encryption.md [hunk 1]
```diff
@@ -116,5 +116,23 @@ servers) and directly store them in your secret management tool.
...
+
+
+To enable smooth rollout of new keys you can use the new option 
+`--rocksdb.encryption-keyfolder` to provide a set of secrets.
+Arangod will then store the master key encrypted with the provided secrets.
+
+```
+$ arangod \
+    --rocksdb.encryption-keyfolder=/mytmpfs/mySecrets \
+    --server.storage-engine=rocksdb
```
Comment (joerg84) [inline 3.7/security-encryption.md:134]: do we need rocksdb here?

Comment (jsteemann) [reply 3.7/security-encryption.md:134]: Not anymore in 3.7/devel

Comment (jsteemann) [inline 3.7/security-encryption.md:134]:
```suggestion
$ arangod \
    --rocksdb.encryption-keyfolder=/mytmpfs/mySecrets
```

File: 3.7/security-encryption.md [hunk 2]
```diff
@@ -116,5 +116,23 @@ servers) and directly store them in your secret management tool.
...
 ## Rotating encryption keys
 
+ArangoDB supports rotating the user supplied encryption at rest key.
+This is implemented via key indirection. The user supplied key is used 
+to encrypt a randomly generated internal master key.
+
 It is possible to change the user supplied encryption at rest key via the
 [HTTP API](http/administration-and-monitoring.html#encryption-at-rest).
+
+
```
Comment (Simran-B) [inline 3.7/security-encryption.md:126]:
```suggestion
```

File: 3.7/security-encryption.md [hunk 3]
```diff
@@ -116,5 +116,23 @@ servers) and directly store them in your secret management tool.
...
+To enable smooth rollout of new keys you can use the new option 
+`--rocksdb.encryption-keyfolder` to provide a set of secrets.
+Arangod will then store the master key encrypted with the provided secrets.
```
Comment (Simran-B) [inline 3.7/security-encryption.md:129]:
```suggestion
_arangod_ will then store the master key encrypted with the provided secrets.
```

**Pull request merged**